ui/src/translations.json

```diff
@@ -242,7 +242,9 @@
     "Total": "Total",
     "sub flow": "Sub-flow",
     "execute": "Execute",
-    "execute new tab": "Execute in new tab",
+    "execute flow behaviour": "Opening of a new execution",
```

Comment:
```suggestion
    "execute flow behaviour": "Execute the flow",
```

otherwise it's too long/verbose

![image](https://github.com/kestra-io/kestra/assets/86264395/98f3b82a-343a-438a-8437-81a1b6f3c447)